lib/components/viewers/RouteRow.js

```diff
@@ -119,7 +119,8 @@ export class RouteRow extends PureComponent {
   }
 
   _onClick = () => {
-    const { isActive, route, setViewedRoute } = this.props
+    const { findRouteIfNeeded, isActive, route, setViewedRoute } = this.props
+    findRouteIfNeeded({ routeId: route.id })
```

Comment:
Going from hover to onClick is causing a bit of a negative user experience, especially on slow connections. The geometry is sometimes not loaded in time to display before the animation fires. Even if it does arrive in time, the delay means that all stops briefly flash unpleasantly.

Reply:
I'm thinking hover can be problematic for a11y, and users with certain assistive technologies may not be able to trigger the behavior otherwise. Maybe implementing a "focus" thing such as in a regular list could help resolve that?

Reply:
I'd be ok with firing the onHover handler on focus

Reply:
Cool. Updated in https://github.com/opentripplanner/otp-react-redux/pull/675/commits/0d581bae5c9cf4b964631312bd70e388bcc88e33.